OSVR-Unity/Assets/OSVRUnity/src/VREye.cs

```diff
@@ -247,20 +249,31 @@ public void CreateSurfaces(uint surfaceCount)
 
                     //distortion
                     bool useDistortion = Viewer.DisplayController.DisplayConfig.DoesViewerEyeSurfaceWantDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
-                    if(useDistortion)
+                    useDistortion |= !displayController.UseRenderManager;
+
+                    if (useDistortion)
                     {
                         //@todo figure out which type of distortion to use
                         //right now, there is only one option, SurfaceRadialDistortion
                         //get distortion parameters
-                        OSVR.ClientKit.RadialDistortionParameters distortionParameters =
-                        Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(
-                        Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
+                        OSVR.ClientKit.RadialDistortionParameters distortionParameters;
+
+                        try
+                        {
+                            distortionParameters = Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
```

Comment:
GetViewerEyeSurfaceRadialDistortion is looking for radial distortion coefficients, which only some of the display descriptors use (depending on whether or not the lenses are radially distorted). HDK1.3/1.4 use polynomial distortion or a mesh. HDK2 uses a mesh.

```C#
bool useDistortion = Viewer.DisplayController.DisplayConfig.DoesViewerEyeSurfaceWantDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
useDistortion |= !displayController.UseRenderManager;
```
Does DoesViewerEyeSurfaceWantDistortion return false? According to the comments it could still return true if any type of distortion is used in the display descriptor, but it doesn't mean that radial distortion coefficients (k1,k2,k3) are available. The call to GetViewerEyeSurfaceRadialDistortion isn't expected to work if those coefficients are not in the display descriptor, so I don't think anything is broken there.

Can you try using a display descriptor that uses those same hard-coded values for k1,k2,k3 and see if you get the same results? Here is an [example](https://github.com/OSVR/OSVR-Core/blob/master/apps/displays/Sensics_zSight_1280_1024_1input.json#L29).